setup.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+
+###############################################################################
+# Copyright (c) 2019, National Research Foundation (Square Kilometre Array)
+#
+# Licensed under the BSD 3-Clause License (the "License"); you may not use
+# this file except in compliance with the License. You may obtain a copy
+# of the License at
+#
+#   https://opensource.org/licenses/BSD-3-Clause
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+
+from setuptools import setup, find_packages
+
+setup(
+    name="scarif",
+    description="Open source tape library control",
+    author="Thomas Bennett",
+    author_email="thomas@ska.ac.za",
+    packages=find_packages(),
+    install_requires=[
+        "serial"
+    ],
+    url='http://ska.ac.za/',
+    scripts=[
+        "scripts/example.py",
```

Comment:
I dont think we should install the example.

Reply:
Agreed. Thanks.